components/renku_data_services/search/core.py

```diff
@@ -88,6 +88,8 @@ async def update_solr(search_updates_repo: SearchUpdatesRepo, solr_client: SolrC
             else:
                 counter = counter + len(entries)
                 await search_updates_repo.mark_processed(ids)
+
+            await solr_client.delete("deleted:true")
```

Comment:
Eike can you please put a comment here explaining what this does. I think you explained it to me. Where we "soft-delete" solr documents during syncing and this cleans up these soft-deleted entities from solr fully. Right? Withtout this we would keep accumulating soft deleted documents in solr?

Reply:
oh sure! Yes, this correct (soft-delete). I also forgot to try-catch it.